yarn.lock

```diff
@@ -1978,6 +1978,11 @@ caller-id@^0.1.0:
   dependencies:
     stack-trace "~0.0.7"
 
+callsite@^1.0.0:
```

Comment:
@marcdumais-work they are both of MIT license, unfortunately callsite does not have Git repo, but it is MIT in the package.json

Reply:
Here you go: https://github.com/tj/callsite

Reply:
Both dependencies look good: MIT licenses and no production dependencies of their own (so no potential that something with inappropriate license would be pulled).